_posts/blog/2018-09-18/2018-09-18-5 -tips-to-make-your-openbazaar-listings-pop.markdown

```diff
@@ -0,0 +1,158 @@
+title: "<span class='post-title-extra'>5 Tips</span> To Make Your OpenBazaar Listings <span class='post-title-extra'>Pop</span>"
+layout: post
+date: '2018-09-18 06:30:00 -0300'
+social_title: '5 Tips To Make Your OpenBazaar Listings POP!'
+social_description: 'With OpenBazaar, the largest decentralized marketplace in the world, you can build your store and listings faster than on any other network and we have some tips that will help your store stand out.'
+social_image: OpenBazaar Store on OpenBazaar.png
+---
+
+_By Monique Boediono, OpenBazaar UX Designer_
+
+Ecommerce is an exciting and flexible way to start your own business. With some ecommerce business models you can even earn money from the comfort of your own home. While there are many opportunities to get started though, there are also many ways to get lost in the crowd. 
+
+![OpenBazaar Store on OpenBazaar](OpenBazaar Store on OpenBazaar.png "OpenBazaar Store on OpenBazaar")
+
+With OpenBazaar, the world's largest decentralized marketplace, you can build your store and listings faster than on any other network and we have some tips that will help your store stand out.
+
+If you have something you’ve been itching to sell on OpenBazaar, here are 5 tips to help make your listings **POP** on the network! 🎈
+
+![Creating a Listing Title on OpenBazaar](Creating Listing Tags on OpenBazaar.gif "Creating a Listing Title on OpenBazaar")
+
+### 1. Write a descriptive title
+
+Your title should give your buyers a clear idea of what you’re selling. When writing a title, try throwing in a few descriptive keywords to give your listing some extra oomph. For example, “hand-crafted pure copper jug with lid” is a lot more interesting and helpful than just “water jug”.
+
+### 2. Take many, many photos 📸
+
+If a picture is worth a thousand words and you can attach up to 30 photos for every listing, that should save you 30,000 words, right? (And way too many headaches.) Take advantage of this.
+
+Photos are arguably the most important part of a listing -- it’s the closest a buyer can come to seeing your product in person. Take clear, high-quality pictures in plenty of light to really let your product shine.
+
+![Simple Listing Image on OpenBazaar](Simple Listing Image on OpenBazaar.jpg "Simple Listing Image on OpenBazaar")
+
+#### Get rid of background clutter
+
+Don’t detract from the star of the show. Stage your product against a plain background. This removes distractions and allows your product to stand out.
+
+> **TIP:** Can’t find a good plain surface anywhere? Textured surfaces can also work well. Place your product on marble counters and wooden coffee tables. Or, just hang up a bedsheet!
+
+#### Give us variety
+
+Show us how your product looks in different angles and settings. Snap some closeups to capture unique and intricate details. Showcase your product in action. If you’re selling a t-shirt, take photos of the shirt on a model so your buyers can see how much cooler it looks on a human being. 
```

Comment:
A product wouldn't be in an angle, but it would be in a setting. You could use "with different angles and in different settings." 

Or "with different angles and settings", a product photo can be with both an angle and a setting.